articles/language/file-structure.md

```diff
@@ -42,7 +42,7 @@ name of the namespace, an open brace `{`, the definitions contained in the
 namespace, and a close brace `}`.
 Namespace names follow the .NET pattern of a sequence of one or more legal
 symbols separated by periods `.`.
-For instance, `MyQuantumStuff` and `Microsoft.Quantum.Canon` are valid
+For instance, `MyQuantumStuff` and `Microsoft.Quantum.Standard` are valid
```

Comment:
The `Microsoft.Quantum.Canon` namespace is still a part of the `Microsoft.Quantum.Standard` package, such that I think this snippet still works?
```suggestion
For instance, `MyQuantumStuff` and `Microsoft.Quantum.Canon` are valid
```